build.xml

```diff
@@ -54,18 +54,19 @@
     <!-- FIXTURES -->
     <target name="fixtures" depends="doctrine">
         <echo message="Loading data fixtures..." />
-        <exec executable="bin/console" checkreturn="true">
-            <arg value="doctrine:fixtures:load" />
-            <arg value="--no-interaction" />
-        </exec>
+        <phingcall target="fixturesRun"/>
```

Comment:
Good. But what is the purpose of target "fixtures" ? It's identical as target "fixturesRun". Just move this echo to fixturesRun.